src/test/java/com/google/api/codegen/testdata/csharp_gapic_resourcenames_library.baseline

```diff
@@ -780,16 +780,16 @@ namespace Google.Example.Library.V1
     public partial class FindRelatedBooksRequest
     {
         /// <summary>
-        /// <see cref="IList{BookName}"/>-typed view over the <see cref="Names"/> resource name property.
+        /// <see cref="ResourceNameList{BookName}"/>-typed view over the <see cref="Names"/> resource name property.
         /// </summary>
-        public IList<BookName> BookNames =>
+        public ResourceNameList<BookName> BookNames =>
```

Comment:
Because to initialise it using Object Initialisation syntax we need access to the Add(IEnumerable<>) method, which is not available on IList. Unfortunately.